celery/backends/redis.py

```diff
@@ -232,11 +232,14 @@ def __init__(self, host=None, port=None, db=None, password=None,
             'max_connections': self.max_connections,
             'socket_timeout': socket_timeout and float(socket_timeout),
             'retry_on_timeout': retry_on_timeout or False,
-            'socket_keepalive': socket_keepalive or False,
             'socket_connect_timeout':
                 socket_connect_timeout and float(socket_connect_timeout),
         }
 
+        # absent in redis.connection.UnixDomainSocketConnection
```

Comment:
dont we need to update tests for this?

Reply:
Done, I've added a check of this case below